tests/functional/placementapi_controller_test.go

```diff
@@ -159,6 +160,7 @@ var _ = Describe("PlacementAPI controller", func() {
 			mariadb.SimulateMariaDBDatabaseCompleted(names.MariaDBDatabaseName)
 			mariadb.SimulateMariaDBAccountCompleted(names.MariaDBAccount)
 			th.SimulateJobSuccess(names.DBSyncJobName)
+			th.SimulateDeploymentAnyNumberReplicaReady(names.DeploymentName, 0)
```

Comment:
now you can use the DeploymentReady() as well as it will automatically set 0 ReadyReplicas if 0 is requested in the Spec